content/en/bits_ai/managing_incidents.md

```diff
@@ -1,46 +1,70 @@
 
+Bits AI simplifies incident management processes, improves collaboration, and provides valuable support for incident responders, making it a beneficial tool for efficient incident resolution.
+
 ## Prerequisites
 
-- Make sure you've enabled the "Mirror all incident Slack channel messages to its incident timeline in real-time" in **[Incident > Settings > Integrations][3] > Slack**. This allows Datadog to generate summaries and postmortems from Slack conversations.
-- The incident channel name must be prefixed with `#incident-`. This enables you to use Bits AI in Slack (`@Datadog`) to ask questions about the incident.
-- Your Datadog account must be connected to Slack. Running a query automatically initiates this process, which can be completed by following the prompts.
+- Your Datadog account must be connected to Slack. Running the `/dd` connect command automatically initiates this process, which can be completed by following the prompts.
+- In **[Incident > Settings > Integrations][3] > Slack**, enable the **"Push Slack channel messages to the incident timeline"** and **"Activate Bits AI features in incident Slack channels for your organization"** toggles. This allows Datadog to ingest Slack conversations into the incident timeline to generate summaries and postmortems. **Note**: a Slack workspace can only have Bits AI activated for one Datadog Organization.   
+- Incident channels you'd like to work with Bits AI must be prefixed with `#incident-`. 
+- To ask Bits AI questions about incidents from any Slack channel, you must invite Bits AI to that channel. Run the `@Datadog` command and follow the instructions on the screen. 
+
+{{< img src="bits_ai/managing_incidents/integration-settings.png" alt="Slack integration settings in Datadog" style="width:90%;">}}
+
+## View incident summaries
+
+When you join an incident channel in Slack, you automatically receive a summary of the incident. The channel must be connected to Incident Management and have at least ten messages. The summary, which is only visible to you, does not persist across reloads, between desktop and mobile apps, or across sessions.
+
+At any time, you can ask for a fresh summary by asking `@DataDog Give me a summary of this incident`. 
 
-{{< img src="bits_ai/integration-settings.png" alt="Slack integration settings in Datadog" style="width:90%;">}}
+## Search across your entire incident history and ask questions
 
-## View incident summaries and schedule updates
+You can ask Bits to find incidents that you're looking for. For instance,
+- `@Datadog How many incidents are currently ongoing?`
+- `@Datadog A customer is unable to check out. Is there an incident?`
+- `@Datadog Show me all Sev-1 incidents that occurred in the past week`
+- `@Datadog Were there any incidents in the past month that impacted the payments service?` 
```

Comment:
```suggestion
- `@Datadog Were there any incidents in the past month that impacted the payments service?` 
```

Reply:
Already using these examples in the following paragraphs

Reply:
Will remove
```suggestion
```